src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/DocumentConversionService.java

```diff
@@ -85,6 +92,36 @@ public byte[] flattenPdfDocument(byte[] document) {
         return document;
     }
 
+    private void flattenAnnotations(PDDocument doc) throws IOException {
```

Comment:
There's a lot of nesting going on here (google arrow anti pattern). 
@al-hmcts FYI

Reply:
Split this method into small method to avoid nesting